CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 Change Log
 ==========
+Version 0.8.1 *(7th March 2022)*
+-------------------------------------------
+- Supports CleverTap iOS SDK `v4.0.0`
+- abstract out notification click callback logic for killed state for Android.
```

Comment:
@piyush-kukadiya Re-word this to -> 
Abstract out notification click callback logic for killed state in Android.